hack/deploy/build-single-manifests.sh

```diff
@@ -5,22 +5,11 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/../..
-STAGE_DIR=$(mktemp -d)
-
-mkdir $STAGE_DIR/base
+CONFIG_ROOT=config/
```

Comment:
Shouldn't this be relative to `SCRIPT_ROOT`? Also the trailing `/` seems superfluous.

```suggestion
CONFIG_ROOT="$SCRIPT_ROOT/config"
```

Also (I know that this falls outside of the scope of this PR, but :shrug:) `SCRIPT_ROOT` is an inaccurate name for the variable on the line above.
This variable points to the root directory of the repo, and not to the directory in which the script is located. This is an (imo incorrect) practice prevalent throughout our codebase and I'd like to take care of it - see my similar comment on another PR: https://github.com/Kong/kubernetes-ingress-controller/pull/1601#discussion_r679263801.

Reply:
Similar to the other comment, the variable name was just already there. I've changed it.

There didn't really seem to be any reason for CONFIG_ROOT since it's static. I forget why I added it and have instead just gone back to using the full relative path literal in the kustomize commands.